src/main/scala/viper/gobra/frontend/info/base/Type.scala

```diff
@@ -84,34 +92,38 @@ object Type {
       decl.methSpecs.isEmpty && decl.predSpec.isEmpty &&
         decl.embedded.isEmpty
     }
+
+    override lazy val toString: String = {
+      if (isEmpty) "interface{}" else "interface{...}"
+    }
   }
 
 
-  case class InternalTupleT(ts: Vector[Type]) extends Type
+  case class InternalTupleT(ts: Vector[Type]) extends PrettyType(s"(${ts.mkString(",")})")
 
   case class InternalSingleMulti(sin: Type, mul: InternalTupleT) extends Type
 
-  case class ImportT(decl: PImport) extends Type
+  case class ImportT(decl: PImport) extends PrettyType(decl.formatted)
 
-  case object SortT extends Type
+  case object SortT extends PrettyType("sort")
```

Comment:
I believe that `"Type"` is preferable to `"sort"` in order to be consistent with the Go docs (e.g. https://golang.org/src/builtin/builtin.go?s=6524:6544#L160)

Reply:
While I think that "Type" is a bad name, I agree that picking the same name as Go is preferrable.